frontend/src/routes/(authenticated)/user/+page.svelte

```diff
@@ -3,14 +3,28 @@
   import { Button, Form, FormError, Input, lexSuperForm } from '$lib/forms';
   import t from '$lib/i18n';
   import { Page } from '$lib/layout';
-  import { notifySuccess } from '$lib/notify';
+  import { _changeUserAccountData } from './+page';
+  import { notifySuccess, notifyWarning } from '$lib/notify';
   import z from 'zod';
+  import { goto } from '$app/navigation';
+  import DeleteUserModal from '$lib/components/DeleteUserModal.svelte';
   import type { PageData } from './$types';
-  import { _changeUserAccountData } from './+page';
+  import { TrashIcon } from '$lib/icons';
+  import {onMount} from 'svelte';
+  import {DialogResponse} from '$lib/components/modals';
 
   export let data: PageData;
-  const user = data.user;
+  $: user = data?.user;
+  $: userid = user?.id;
+  let deleteModal: DeleteUserModal;
 
+  async function openDeleteModal(): Promise<void> {
+    let {response} = await deleteModal.open(userid);
+    if (response == DialogResponse.Submit){
+      notifyWarning($t('account_settings.delete_success'))
+    }
+    await goto('/logout');
```

Comment:
we don't want to logout if the user cancels the delete do we?

Reply:
Oops, Good point! It should be fixed now